README.md

```diff
@@ -381,14 +379,8 @@ an adoption helper, avoid using this for new projects.
 *Black* standardizes most numeric literals to use lowercase letters for the
 syntactic parts and uppercase letters for the digits themselves: `0xAB`
 instead of `0XAB` and `1e10` instead of `1E10`. Python 2 long literals are
-styled as `2L` instead of `2l` to avoid confusion between `l` and `1`. In
-Python 3.6+, *Black* adds underscores to long numeric literals to aid
-readability: `100000000` becomes `100_000_000`.
-
-For regions where numerals are grouped differently (like [India](https://en.wikipedia.org/wiki/Indian_numbering_system)
-and [China](https://en.wikipedia.org/wiki/Chinese_numerals#Whole_numbers)),
-the `-N` or `--skip-numeric-underscore-normalization` command line option
-makes *Black* preserve underscores in numeric literals.
+styled as `2L` instead of `2l` to avoid confusion between `l` and `1`.
+In Python 3.6 and later, *Black* preserves underscores in numeric literals.
```

Comment:
Maybe add "unchanged"? Otherwise, I'm not sure it's worth mentioning something Black *doesn't* change.

Reply:
Hm, fair enough. I'll just remove this sentence.